docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -43,9 +45,23 @@ contents:
    access-control.name=file
    security.config-file=etc/rules.json
 
+If the config should be loaded via the rest endpoint ``http://trino-test/config`` and
+is wrapped into a JSON object and available via the ``data`` key ``etc/access-control.properties``
+should look like this:
+
+.. code-block:: text
+
+   access-control.name=rest
+   security.config-url=http://trino-test/config
+   security.json-pointer=/data
```

Comment:
Could you add tests using `security.json-pointer`? I'm not sure if this property is needed in this initial PR. Does your environment require this property?

Reply:
Two tests added to cover json-pointer case - see TestRestBasedAccessControl and TestRestBasedSystemAccessControl

Reply:
What about "Does your environment require this property?"

Reply:
yes, we use it